pkg/apis/ocs/v1/storagecluster_types.go

```diff
@@ -14,14 +15,16 @@ import (
 
 // StorageClusterSpec defines the desired state of StorageCluster
 type StorageClusterSpec struct {
-	ManageNodes  bool   `json:"manageNodes,omitempty"`
-	InstanceType string `json:"instanceType,omitempty"`
+	ManageNodes   bool                 `json:"manageNodes,omitempty"`
+	InstanceType  string               `json:"instanceType,omitempty"`
+	LabelSelector metav1.LabelSelector `json:"labelSelector,omitempty"`
```

Comment:
What is this used for?

Reply:
To specify custom label to run OCS apart from the default OCS label.

Reply:
Why are you not using NodeAfiinity directly?

Reply:
Because we don't want to disturb the current UI workflow and this seems reasonalble. Also, the ocs label (NodeAffinityKey) is hardcoded in ```reconcileNodeTopologyMap``` and this could help in handling that.